plugins/tileacq.py

```diff
@@ -752,17 +753,31 @@ def acquire(self, dlg):
                 logging.info("Computing big image out of %d images", len(da_list))
                 das_registered = stitching.register(da_list)
 
+                # Select weaving method
+                # On a Delphi and Secom system the mean weaver gives the best result since it
```

Comment:
No, that should be the opposite: On the SPARC, the ebeam exposition tends to have limited effects on the sample, so the mean gives best effects. On the SECOM/DELPHI, the ebeam tends to bleach the optical images, and may cause large carbon deposition making dark pattern. So in such case, it preferable to use the data from only the first acquisition.